src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -334,29 +326,8 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
 
       try {
         const parsedValue = parseJSONWithNumericString(args.value) as ParsedTokenConfig;
-
         const l1Token = args.key;
 
-        // For now use the presence of `uba` or `rateModel` to decide which configs to parse.
-        if (parsedValue?.uba !== undefined) {
```

Comment:
Note - assuming we also want to remove ConfigStore support for the UBA, we should be sure to also flush the config in the onchain update(s).

Reply:
agreed

Reply:
Have recorded this in ACX-1846 :+1: